tuxemon/core/components/monster.py

```diff
@@ -274,26 +273,67 @@ def load_from_db(self, slug):
         **Examples:**
 
         >>> bulbatux = Monster()
-        >>> bulbatux.load_from_db("Bigfin")
-
+        >>> bulbatux.load("Bigfin")
         """
 
         # Look up the monster by name and set the attributes in this instance
-        results = monsters.lookup(slug)
+        results = monsters_db.lookup(slug)
 
         if results is None:
-            logger.error("monster {} is not found".format(slug))
+            logger.error("monster {} not found".format(slug))
             raise RuntimeError
 
         self.slug = results["slug"]                             # short English identifier
-        self.name = trans(results["name_trans"])                # will be locale string
-        self.description = trans(results["description_trans"])  # will be locale string
-        self.shape = results.get("shape", "landrace").lower()
-        types = results.get("types")
-        if types:
-            self.type1 = results["types"][0].lower()
-            if len(types) > 1:
-                self.type2 = results["types"][1].lower()
+        self.name = T.translate(results["slug"])                # translated name
+        self.description = T.translate("{}_description".format(results["slug"]))  # translated description
+        self.category = T.translate(results["category"])        # translated category
+
+        self.hp = results["hp_base"]
```

Comment:
I'll try to compare my branch with the `development` branch by hand then, because I just performed a standard rebase. I had no intention of overwriting your work, apologies.

Reply:
Sorry for all the pain caused by that rebase I asked for...

Getting there, we don't need these:
  "sprites": {
    "battle1": "gfx/sprites/battle/aardart-front.png",
    "battle2": "gfx/sprites/battle/aardart-back.png",
    "menu1": "gfx/sprites/battle/aardart-menu01.png",
    "menu2": "gfx/sprites/battle/aardart-menu02.png"
  },

You're also changing
"types": ["earth"],
to 
"types": ["Earth"],

It shouldn't break anything though.